docs/dashboard.md

```diff
@@ -59,3 +59,15 @@ The sortable table is created in plain JS, while the charts are generated using
 - The "docker-compose.yml" file defines the services and volumes required to run the application using Docker Compose.
 - It is a separate compose file from the main one, designed to quickly bootstrap the dashboard, reduce complexity in the main file, and improve maintainability.
 - The "app" service has a condition `service_completed_successfully`, which means the Git script (dep-checker.sh) needs to complete successfully before running the "app" service.
+
+## Potential Issues
+
+### Dubious Ownership
+
+- After running `make start-dashboard` due to unknown circumstances (It may potentially be a linux specific issue) it can sometimes output the following error: `fatal: detected dubious ownership in repository at REPO_DIRECTORY`
```

Comment:
Thanks @DanielMurray97 best add directly the config on the script to avoid breaking `commits_info.json` file